php/class-customize-snapshot-manager.php

```diff
@@ -1125,6 +1125,11 @@ public function handle_update_snapshot_request() {
 			) );
 		}
 
+		/**
+		 * Add any additional checks before saving snapshot.
+		 */
+		do_action( 'customize_snapshot_save_before', $this->snapshot, $this->customize_manager );
```

Comment:
Since this action is specific to snapshots, I think `$this->customize_manager` should be replaced with just `$this`, the `Customize_Snapshot_Manager`.